content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst

```diff
@@ -0,0 +1,356 @@
+========================
+GS1 barcode nomenclature
+========================
+
+`GS1 nomenclature <https://www.gs1us.org/>`_ consolidates multiple pieces of information in a single
+barcode. Each piece needs to follow a specific barcode pattern—which is a defined format of numbers,
+letters, special characters, and character length—to ensure proper interpretation of the barcode. By
+scanning the barcode on an unopened box, GS1 nomenclature can identify the product, lot number,
+number of units contained, and more.
+
+.. seealso::
+   - `All GS1 barcodes <https://www.gs1.org/standards/barcodes/application-identifiers>`_
+   - :ref:`Odoo's default GS1 rules <barcode/operations/default-gs1-nomenclature-list>`
+   - :ref:`Why's my barcode not working? <barcode/operations/troubleshooting>`
+
+.. _barcode/operations/set-up-barcode-nomenclature:
+
+Set up barcode nomenclature
+===========================
+
+To use GS1 nomenclature, navigate to the :menuselection:`Inventory app --> Configuration -->
+Settings`. Then under the :guilabel:`Barcode` section, check the :guilabel:`Barcode Scanner` box.
+Next, select :menuselection:`Barcode Nomenclature --> Default GS1 Nomenclature` from the default
+barcode nomenclature options.
+
+.. image:: gs1_nomenclature/setup-gs1-nomenclature.png
+   :align: center
+   :alt: Choose GS1 from dropdown and click the internal link to see the list of GS1 rules.
+
+To view and edit a list of GS1 *rules* and *barcode patterns* Odoo supports by default, click the
+:guilabel:`➡️ (External link)` icon to the right of the :guilabel:`Barcode Nomenclature` selection.
+
+Opening the pop-up table provides an editable view of GS1 :guilabel:`Rule Names` available in Odoo.
+The table contains all the information that can be condensed with a GS1 barcode, along with the
+corresponding :guilabel:`Barcode Pattern`.
+
+.. tip::
+   After setting GS1 as the barcode nomenclature, :menuselection:`Barcode Nomenclatures` can also be
+   accessed by first enabling :ref:`developer mode <developer-mode>`. Navigate to
+   :menuselection:`Inventory app --> Configuration --> Barcode Nomenclatures` and finally, select
+   :guilabel:`Default GS1 Nomenclature`.
+
+.. _barcode/operations/create-GS1-barcode:
+
+Use GS1 barcode
+===============
+
+To build GS1 barcodes in Odoo, combine multiple pieces of information using the specified barcode
```

Comment:
> To build GS1 barcodes in Odoo

This implies Odoo will build the barcodes, meaning, we give Odoo some input data, and Odoo will spit out output data in the form of barcodes.

That doesn't appear to be what's happening? Or at least...that's not clear to me in the text.

It seems like there are rules/conventions to follow, and that's fine, however, I don't see clear instructions or a clear conclusion that Odoo builds barcodes in this section...